api/specs.pb.go

```diff
@@ -2781,6 +2821,14 @@ func (m *ContainerSpec) Size() (n int) {
 	if m.PidsLimit != 0 {
 		n += 2 + sovSpecs(uint64(m.PidsLimit))
 	}
+	if len(m.Sysctls) > 0 {
```

Comment:
~Same here; this check looks redundant~

edit: nevermind. Didn't notice I was looking at generated code ("Code generated by protoc-gen-gogo. DO NOT EDIT.") :blush: